cetmix_tower_yaml/static/description/index.html

```diff
@@ -435,32 +460,72 @@ <h2><a class="toc-backref" href="#toc-entry-7">17.0.1.0.1 (2025-05-05)</a></h2>
 </ul>
 </div>
 <div class="section" id="section-5">
+<<<<<<< HEAD
 <h2><a class="toc-backref" href="#toc-entry-8">17.0.1.0.0</a></h2>
 <p>Release for Odoo 17.0</p>
 </div>
 </div>
 <div class="section" id="bug-tracker">
 <h1><a class="toc-backref" href="#toc-entry-9">Bug Tracker</a></h1>
+=======
+<h3><a class="toc-backref" href="#toc-entry-8">16.0.1.0.3 (2025-05-09)</a></h3>
+<ul class="simple">
+<li>Bugfixes: Non-critical issues and performance improvements. (4663)</li>
+</ul>
+</div>
+<div class="section" id="section-6">
+<h3><a class="toc-backref" href="#toc-entry-9">16.0.1.0.2 (2025-04-30)</a></h3>
+<ul class="simple">
+<li>Features: User groups are visible without developer mode. (4642)</li>
+</ul>
+</div>
+<div class="section" id="section-7">
+<h3><a class="toc-backref" href="#toc-entry-10">16.0.1.0.1 (2025-04-21)</a></h3>
+<ul class="simple">
+<li>Features: Export additional fields for shortcuts, variables and
+options. Add action menu to export keys/secrets. (4602)</li>
+</ul>
+</div>
+<div class="section" id="section-8">
+<h3><a class="toc-backref" href="#toc-entry-11">16.0.1.0.0</a></h3>
+<p>Release for Odoo 16.0</p>
+</div>
+</div>
+<div class="section" id="bug-tracker">
+<h2><a class="toc-backref" href="#toc-entry-12">Bug Tracker</a></h2>
+>>>>>>> f8b8067 ([FIX]  cetmix_tower_yaml: bypass SSH check)
 <p>Bugs are tracked on <a class="reference external" href="https://github.com/cetmix/cetmix-tower/issues">GitHub Issues</a>.
 In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us to smash it by providing a detailed and welcomed
 <a class="reference external" href="https://github.com/cetmix/cetmix-tower/issues/new?body=module:%20cetmix_tower_yaml%0Aversion:%2017.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**">feedback</a>.</p>
 <p>Do not contact contributors directly about support or help with technical issues.</p>
 </div>
 <div class="section" id="credits">
+<<<<<<< HEAD
 <h1><a class="toc-backref" href="#toc-entry-10">Credits</a></h1>
 <div class="section" id="authors">
 <h2><a class="toc-backref" href="#toc-entry-11">Authors</a></h2>
+=======
```

Comment:
This is left from merge conflict resolve, you must resolve the conflicts properly